base_local_planner/src/speed_limiters/obstacle_speed_limiter.cpp

```diff
@@ -167,76 +167,79 @@ double ObstacleSpeedLimiter::getBearingToObstacle(const costmap_2d::ObstructionM
   return atan2(obs.y, obs.x);
 }
 
-    ObstacleSpeedLimiter::LinearSpeedLimiterResult ObstacleSpeedLimiter::calculateAllowedLinearSpeed(const costmap_2d::ObstructionMsg& obs)
-    {
-      LinearSpeedLimiterResult result;
+ObstacleSpeedLimiter::LinearSpeedLimiterResult ObstacleSpeedLimiter::calculateAllowedLinearSpeed(const costmap_2d::ObstructionMsg& obs)
```

Comment:
I can't tell what is actually different because of the formatting change.  What lines are different?

Reply:
There are not differences, I removed a tab from each line

Reply:
Actually the difference are 206-209 of the incoming changes